BCnEnc.Net/Shared/DdsFile.cs

```diff
@@ -358,12 +363,24 @@ public DxgiFormat DxgiFormat
 				{
 					switch (dwFourCc)
 					{
-						case 0x31545844U:
+						case Dxt1:
```

Comment:
Dds format correct reading seems to be handled here but is the dwflags written to when encoding?

Reply:
I just expanded the already existing code that uses dwFourCc. I assume that if it worked before, it should now as well. I will check that again.

Reply:
If you'd like to support even more, there's some undocumented ones I've seen used. For example BC4U, from the Intel Texture Plugin for Photoshop.
https://github.com/GameTechDev/Intel-Texture-Works-Plugin/blob/6907bb8c18fbe5a7879d4f5bd53b9ef507a79456/3rdParty/DirectXTex/DirectXTex/DDS.h#L84

I also found these in the DXTex code:
https://github.com/microsoft/DirectXTex/blob/234eee8a019f4a53adef5b86c90efe9a7f5f3111/DirectXTex/DirectXTexDDS.cpp#L73

Reply:
You were right, I expanded the InitializeCompressed method to properly set the dwPixelFormat. As with all the other occurrences I also added BC4 and BC5 here.